sklearn/model_selection/_prediction.py

```diff
@@ -0,0 +1,462 @@
+from inspect import signature
+import numbers
+
+import numpy as np
+from joblib import Parallel, delayed
+
+from ._split import check_cv
+from ._split import StratifiedShuffleSplit
+
+from ..base import clone
+from ..base import BaseEstimator
+from ..base import ClassifierMixin
+from ..base import MetaEstimatorMixin
+from ..metrics import balanced_accuracy_score
+from ..metrics import roc_curve
+from ..preprocessing import LabelEncoder
+from ..utils import check_array
+from ..utils import _safe_indexing
+from ..utils.multiclass import check_classification_targets
+from ..utils.multiclass import type_of_target
+from ..utils.validation import check_is_fitted
+
+
+class CutoffClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
+    """Decision threshold calibration for binary classification.
+
+    Estimator that calibrates the decision threshold (cutoff point) that is
+    used for prediction. The methods for picking cutoff points make use of
```

Comment:
By "The methods" you mean the CV procedure? It slightly ambiguous with the `method` parameter.